app/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { render } from 'react-dom';
 import { Provider } from 'react-redux';
 import { Router } from 'react-router';
+import { hashHistory } from 'react-router';
```

Comment:
@borNfreee You can't use `browserHistory` in electron apps. Because routes matching will rely on file path.

```
Warning: [react-router] Location "/Users/xxx/electron-react-boilerplate/app/hot-dev-app.html" did not match any routes
```